app/controllers/devise/registrations/registrations_controller.rb

```diff
@@ -0,0 +1,167 @@
+class Devise::Registrations::RegistrationsController < DeviseController
```

Comment:
If we make this file inherit from `Devise::RegistrationsController`, we only need to overwrite the stuff that we actually wanna change: that's the `sign_up` method only right? so the whole file would look like this:

```rb
class Devise::Registrations::RegistrationsController < Devise::RegistrationsController
  private

  def sign_up(resource_name, resource)
    sign_in(resource_name, resource)
    redirect_to new_user_confirmation_path
  end
end
```

Is there a reason why you decided to copy the entire content of [Devise::RegistrationController](https://github.com/heartcombo/devise/blob/master/app/controllers/devise/registrations_controller.rb)? Maybe I'm missing something here.

Reply:
I changed the Registrations Controller to inherit from Devise, but after carefully checking the original code from Devise I realized that the only method we needed to change is `after_sign_up_path`.